vkext/vkext.cpp

```diff
@@ -255,14 +255,14 @@ void write_buff_long(long x) {
   if (unlikely (wptr + 25 > buff + BUFF_LEN)) {
     flush_buff();
   }
-  wptr += sprintf(wptr, "%ld", x);
+  wptr += snprintf(wptr, 25, "%ld", x);
 }
 
 void write_buff_double(double x) {
   if (unlikely (wptr + 100 > buff + BUFF_LEN)) {
     flush_buff();
   }
-  wptr += sprintf(wptr, "%lf", x);
+  wptr += snprintf(wptr, 100, "%f", x);
```

Comment:
why not "%lf" as before?
```suggestion
  wptr += snprintf(wptr, 100, "%lf", x);
```

Reply:
Because php has **snprintf.h** with 

```
#ifdef snprintf
#undef snprintf
#endif
#define snprintf ap_php_snprintf
```

If we use "lf" it will throw an exception:
> PHP Fatal error:  Illegal length modifier specified 'f' in s[np]printf call

I found some ~~legacy~~ info about that: https://externals.io/message/14592

But we can use just "f" and it will be work fine, because when arguments of type float are passed as variadic parameters, such arguments are implicitly converted to type double.